manifests/init.pp

```diff
@@ -1,35 +1,11 @@
-# Author::    Liam Bennett (mailto:liamjbennett@gmail.com)
-# Copyright:: Copyright (c) 2014 Liam Bennett
-# License::   MIT
-
-# == Define: dotnet
-#
-# Module to install the Microsoft .NET framework on windows
-#
-# === Requirements/Dependencies
-#
-# Currently reequires the puppetlabs/stdlib module on the Puppet Forge in
-# order to validate much of the the provided configuration.
-#
-# === Parameters
-#
-# [*ensure*]
-# Control the state of the .NET installation
-#
-# [*version*]
-# The version of .NET to be managed
-#
-# [*package_dir*]
-# If installing .NET from a directory or a mounted network location then this is that directory
-#
-# === Examples
-#
-#  Installing .NET 4.5
-#
-#    dotnet { 'dotnet45':
-#      version => '4.5',
-#    }
-#
+# @summary Module to install the Microsoft .NET framework on Windows.
+# @param version The version of .NET to be managed.
```

Comment:
"Separate comment sections (such as @summary, @example, or the @param list) with a blank comment line (that is, a # with no additional content), to improve readability."
https://puppet.com/docs/puppet/5.5/puppet_strings_style.html